gradle.properties

```diff
@@ -29,6 +29,11 @@ POM_SETTING_LICENCE_URL=http://www.apache.org/licenses/LICENSE-2.0.txt
 POM_SETTING_LICENCE_DIST=repo
```

Comment:
Don't we need to bump up the artefact version for _tagging on git_ and _publishing_ the snapshot?

Reply:
Yes, we can do that here but I prefer to have this on separate PRs for better tracking since releases were not always done after every PR https://github.com/onaio/kujaku/pulls?q=is%3Apr+sort%3Aupdated-desc+is%3Aclosed+release